src/internal/m365/collection/drive/delta_tree_test.go

```diff
@@ -168,30 +181,37 @@ func (suite *DeltaTreeUnitSuite) TestFolderyMcFolderFace_SetFolder() {
 			defer flush()
 
 			tree := test.tree(t, drive())
+			folder := test.folder()
 
-			err := tree.setFolder(
-				ctx,
-				test.parentID,
-				test.id,
-				test.name,
-				test.isPackage)
+			err := tree.setFolder(ctx, folder)
 			test.expectErr(t, err, clues.ToCore(err))
 
 			if err != nil {
 				return
 			}
 
-			result := tree.folderIDToNode[test.id]
+			result := tree.folderIDToNode[ptr.Val(folder.GetId())]
 			require.NotNil(t, result)
-			assert.Equal(t, test.id, result.id)
-			assert.Equal(t, test.name, result.name)
-			assert.Equal(t, test.isPackage, result.isPackage)
 
-			_, ded := tree.tombstones[test.id]
+			var (
+				expectID        = ptr.Val(folder.GetId())
+				expectName      = ptr.Val(folder.GetName())
+				expectIsPackage = folder.GetPackageEscaped() == nil
+				resultID        = ptr.Val(folder.GetId())
+				resultName      = ptr.Val(folder.GetName())
+				resultIsPackage = folder.GetPackageEscaped() == nil
```

Comment:
looks like copy/paste error here
```suggestion
				resultID        = ptr.Val(result.GetId())
				resultName      = ptr.Val(result.GetName())
				resultIsPackage = result.GetPackageEscaped() == nil
```

Reply:
big oof